core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -114,6 +117,7 @@
     private Executor blockingTaskExecutor = CommonPools.blockingTaskExecutor();
     private MeterRegistry meterRegistry = Metrics.globalRegistry;
     private String serviceLoggerPrefix = DEFAULT_SERVICE_LOGGER_PREFIX;
+    private Consumer<RequestLog> accessLogWriter = AccessLogWriters.common();
```

Comment:
How about using disabled() as a default? If not, it will write access logs with the root logger.